### PR TITLE
Fixes for pbtestkit logging

### PR DESCRIPTION
**Pull request by mpkocher**

### CODE REVIEW

**Comment by mpkocher** [comment]
@natechols can you verify that expected behavior is observed?

**Comment by natechols** [comment]
Sort of - it's missing the level/timestamp prefix in the log messages output by the testkit code (as opposed to the pbsmrtpipe job itself).  I'm pretty sure these were prefixed before.

File: pbsmrtpipe/testkit/runner.py [hunk 1]
```diff
@@ -68,6 +70,7 @@ def _write_xunit_output(test_cases, result, output_xml, job_id):
         f.write(str(jenkins_xml))
     
     log.info("Completed running {t} tests. {s} skipped, {e} errors, and {n} failed tests.".format(e=nerrors, n=nfailed_tests, s=nskipped, t=len(xsuite)))
```
Comment (natechols) [inline pbsmrtpipe/testkit/runner.py]: This log message seems to be getting swallowed with these changes.

**Comment by mpkocher** [comment]
I believe there's too much mixing up of stdout and logging that is creating friction here. The use of stderr on the pbcore PbToolRunner interface is also adding some inconsistency in the system.

I would propose migrating to a model that has minimal high level stdout of the application and a log file to have details of the time stamps and log levels. If you want to get the log description of the application when it's running, then you , than tail -f the file can be used.

Does this work for you?

File: pbsmrtpipe/testkit/runner.py (hunk 1 above)
Comment (mpkocher) [inline pbsmrtpipe/testkit/runner.py]: It's showing up for me, but I am running a local version of pbcommand with some mods that might be creating the differences.

```[INFO] 2016-01-12 01:33:39,840Z Completed running 26 tests. 0 skipped, 0 errors, and 0 failed tests.```

**Comment by natechols** [comment]
I would prefer to err on the side of the verbose (with a --quiet option or similar available), at least including the task processing information shown currently.  However, I think it's more important that failures be logged appropriately than normal behavior.

**Comment by mpkocher** [comment]
I agree with the stderr comments and the error handling this is another source of inconsistency. 

Some tools are pushing errors to stderr or stdout, some are logging the errors, some are pushing to stderr and logging. 

When the log is setup,  a handler to stderr with level of `logging.ERROR` and a minimal format `(message)s` string can be setup. This will catch all errors and make sure they're routed to stderr. This would remove the need for any tool to make hardcoded calls `sys.stderr`

**Pull request merged**